components/list/list-item-generic-layout.js

```diff
@@ -445,16 +445,10 @@ class ListItemGenericLayout extends RtlMixin(LitElement) {
 
 	_onKeydown(event) {
 		if (!this.gridActive) return;
+
 		let node = null;
 		let preventDefault = true;
 		switch (event.keyCode) {
-			case keyCodes.ENTER:
-			case keyCodes.SPACE:
-				node = getComposedActiveElement();
-				node.click();
-				// mouseup event is necessary for components like d2l-dropdown-more that control state by listening to this event
-				node.dispatchEvent(new MouseEvent('mouseup', { composed: true }));
```

Comment:
Entering this `ENTER`/`SPACE` case meant that we were preventing default and stopping propagation even when these keys were pressed for dropdowns/menus in the secondary actions slot, which previously required this fake `mouseup` event hack so dropdowns would open as expected. So eliminating this case logic means we don't need this hack anymore.